crates/libcgroups/src/systemd/manager.rs

```diff
@@ -363,11 +377,11 @@ impl CgroupManager for Manager {
     }
 
     fn freeze(&self, state: FreezerState) -> Result<()> {
-        todo!();
+        self.fs_manager.freeze(state)
```

Comment:
Have you tested this? I'm assuming without a previous implementation there are no test cases for this.

Reply:
I am reusing the cgroup v2 manager here (the systemd dbus interface does not have support for the freezer), so there are tests for it.